cheatsheets/Abuse_Case_Cheat_Sheet.md

```diff
@@ -13,17 +13,15 @@ In order to build a secure application, from an pragmatic point of view, it is i
 
 # Objective
 
-The objective of this cheat sheet is to provide an explanation of what an **Abuse Case** is, why abuse cases are important, and to provide a pragmatic approach to builing abuse cases for tracking during feature implementation of an application. This cheat sheet applies regardless of the project methodology used (waterfall or agile).
+The objective of this cheat sheet is to provide an explanation of what an **Abuse Case** is, why abuse cases are important, and to provide a pragmatic approach to builing abuse cases for tracking during feature implementation of an application. This cheat sheet applies regardless of the project methodology used (Waterfall or Agile).
 
 **Important note about this Cheat Sheet:**
 
-```
-The main objective is to provide a pragmatic approach in order to allow a company or a project
-team to start building and handling the list of abuse cases and then customize the elements
-proposed to its context/culture in order to, finally, build its own method.
-
-This cheat sheet can be seen as a getting-started tutorial.
-```
+> The main objective is to provide a pragmatic approach in order to allow a company or a project
```

Comment:
Just a small note for you to know. If you want the markdown generated from this code to be aligned as you expect it to be, you need to add new lines between every line. If you want we can discuss this on Slack.